### ISSUE TITLE
Questions on guide one "masking and padding with Keras"

### ISSUE THREAD

**Issue by bobbyyan0** [keras-team/keras-io#139]
In https://render.githubusercontent.com/view/ipynb?commit=39b8a76442ef99da623169aaece32898aeea077a&enc_url=68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f6b657261732d7465616d2f6b657261732d696f2f333962386137363434326566393964613632333136396161656365333238393861656561303737612f74662f6d61736b696e675f616e645f70616464696e672e6970796e62&nwo=keras-team%2Fkeras-io&path=tf%2Fmasking_and_padding.ipynb&repository_id=261861733&repository_type=Repository#Writing-layers-that-need-mask-information.

1. Should the line `inputs_sum = tf.reduce_sum(inputs * broadcast_float_mask, axis=1, keepdims=True)` be `inputs_sum = tf.reduce_sum(inputs_exp, axis=1, keepdims=True)`?

2. What does the second argument mean in `y = model(np.random.randint(0, 10, size=(32, 100)), np.random.random((32, 100, 1)))`?

**Comment by sachinprasadhs** [comment]
Hi, 

The link which you have mentioned above seems to be broken.

Could you please provide the code which you are referring to. 

For details on model, please refer the document here https://keras.io/api/models/model/
